<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentWindow</class>
 <widget class="QWidget" name="AgentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>740</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>740</width>
     <height>400</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Добавить соискателя</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/exit_right.svg</normaloff>:/icons/exit_right.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="employeeNameLineEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="employeeSurnameLineEdit">
         <property name="placeholderText">
          <string>Фамилия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="employeePatronymicLineEdit">
         <property name="placeholderText">
          <string>Отчество</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="employeeJobTimeLineEdit">
         <property name="placeholderText">
          <string>Стаж</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="employeePhoneLineEdit">
         <property name="placeholderText">
          <string>Телефон</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="addEmployeeButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Добавить работодателя</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="employerNameLineEdit">
         <property name="placeholderText">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="employerSurnameLineEdit">
         <property name="placeholderText">
          <string>Фамилия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="employerPatronymicLineEdit">
         <property name="placeholderText">
          <string>Отчество</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="employerINNLineEdit">
         <property name="placeholderText">
          <string>ИНН</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="employerPhoneLineEdit">
         <property name="placeholderText">
          <string>Телефон</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="addEmployerButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Удалить работодателя/соискателя</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="Employee">
        <attribute name="title">
         <string>Соискатели</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QTableWidget" name="empleeTable">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>id</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Фамилия</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Имя</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Отчество</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Телефон</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Стаж</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="deleteEmplee">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/cross_circle.svg</normaloff>:/icons/cross_circle.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Employer">
        <attribute name="title">
         <string>Работодатели</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTableWidget" name="emplerTable">
           <column>
            <property name="text">
             <string>id</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Фамилия</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Имя</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Отчество</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Вакансия</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ИНН</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Телефон</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="deleteEmpler">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/cross_circle.svg</normaloff>:/icons/cross_circle.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="refreshEmpls">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/refresh_alt.svg</normaloff>:/icons/refresh_alt.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Верификация</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetActs">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>15</y>
       <width>611</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="verifyActButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icons/box_add.svg</normaloff>:/icons/box_add.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteAct">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icons/cross_circle.svg</normaloff>:/icons/cross_circle.svg</iconset>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
